docs/modules/mongodb.md

```diff
@@ -0,0 +1,78 @@
+# MongoDB
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for MongoDB.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the MongoDB module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/mongodb
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a MongoDB container](../../modules/mongodb/mongodb.go)
```

Comment:
Do we want to share the entire file or just the blocks related to the creation of the container?